### ISSUE TITLE
Auto suggest subs from command line r/...

### ISSUE THREAD

**Issue by jewel-andraia** [honestbleeps/Reddit-Enhancement-Suite#1498]
Prefer subscribed subs

(requested verbally)

**Comment by 3v1n0** [comment]
Yeah, please that's quite a MUST-have feature for a such awesome tool.

It would great to include also some fuzzy-search (à la SublimeText command palette :))

**Comment by hidde-jan** [comment]
This is something that I'm still interested in. I've had a look at the commandLine code, and it seems that it would require quite some changes to implement, or you would need to hijack the getTip function on `r/<subreddit>` to do all kinds of shenanigans, like binding eventListeners en performing ajax requests.

Does anyone of the maintainers have a preference for how one might implement this?
